browser-test/src/trusted_intermediary.test.ts

```diff
@@ -877,9 +877,8 @@ test.describe('Trusted intermediaries', () => {
       await applicantQuestions.clickEdit()
       await applicantQuestions.answerNumberQuestion('5')
       await applicantQuestions.clickNext()
-      await validateToastMessage(page, 'may qualify')
+      await applicantQuestions.expectMayBeEligibileHeadingToBeVisible()
```

Comment:
**suggestion:** (non-blocking):  What would you think of checking here that the TI-specific alert message is shown?  ("Your client may be eligible...")

Reply:
I don't think it's necessary. There's a self-contained class that calculates the message which I have unit tests already covering.